content/blog/2016/2016-09-06-pipeline-at-jenkins-world.adoc

```diff
@@ -0,0 +1,267 @@
+---
+layout: post
+title: "Pipeline at Jenkins World 2016"
+tags:
+- event
+- JenkinsWorld
+author: rtyler
+---
+
+[NOTE]
+--
+This is a guest post by link:https;//github.com/rtyler[R. Tyler Croy], who is a
+long-time contributor to Jenkins and the primary contact for Jenkins project
+infrastructure. He is also a Jenkins Evangelist at
+link:http://cloudbees.com[CloudBees, Inc.]
+--
+
+image:/images/conferences/Jenkins-World_125x125.png[Jenkins World, role=right]
+
+
+I have been heavily using link:/doc/pipeline[Jenkins Pipeline] for just about
+every Jenkins-related project I have touched over the past year. Whether I am
+building and publishing link:https://github.com/jenkins-infra/ircbot[Docker
+containers], link:https://github.com/jenkins-infra/jenkins-infra[testing
```

Comment:
I usually review with the `View` tab anyways, but I'll add this to the styleguide and start using it with my next post